pkg/backend/net/net.go

```diff
@@ -270,7 +274,7 @@ func (b Backend) do(ctx context.Context, trace types.Trace, req *http.Request) (
 		return res.resp.Header.Get("Content-Type"), body, nil
 
 	case <-ctx.Done():
-		trace.ObserveOutDuration(time.Now().Sub(t0))
+		trace.ObserveOutDuration(time.Now().Sub(t0), b.cluster)
```

Comment:
I am not sure we should count cancelled and timed-out requests. (And these are the ones that fall in here)

Reply:
@gksinghjsr , @azhiltsov , @avereha - what do you think on this comment?